sql/testdata/default

```diff
@@ -0,0 +1,72 @@
+statement error incompatible column type and default expression: INT vs bool
+CREATE TABLE t (a INT PRIMARY KEY DEFAULT false)
+
+statement ok
+CREATE TABLE t (
+  a INT PRIMARY KEY DEFAULT 42,
+  b TIMESTAMP DEFAULT now(),
+  c FLOAT DEFAULT random()
+)
+
+query TTTT colnames
+SHOW COLUMNS FROM t
+----
+Field Type      Null Default
+a     INT       true 42
+b     TIMESTAMP true now()
+c     FLOAT     true random()
+
+statement ok
+INSERT INTO t VALUES (DEFAULT, DEFAULT, DEFAULT)
+
+query IBB
+SELECT a, b >= now(), c >= 0.0 FROM t
```

Comment:
Heh, that's a very good question. I'm not sure what is going on, but obviously this is wrong.